bigbluebutton-html5/imports/api/users-settings/server/methods/addUserSettings.js

```diff
@@ -46,7 +45,6 @@ const currentParameters = [
   'bbb_preferred_camera_profile',
   'bbb_enable_screen_sharing',
   'bbb_enable_video',
-  'bbb_enable_video_stats',
```

Comment:
Note to self -  remove from the documentation

Reply:
Did it! https://github.com/bigbluebutton/bigbluebutton.github.io/commit/0a5fbf9b395946aaddc9deafb84d29f1d3f21b05